plug-ins/groups/class_thief.cpp

```diff
@@ -791,10 +803,11 @@ class PushMovement : public ExitsMovement {
 
         percent  = number_percent( ) + (IS_AWAKE(ch) ? 30 : -30);
         percent += ch->can_see( actor ) ? 20 : 0;
+	percent += skill_level_bonus(*gsn_push, ch);    
 
```

Comment:
percent is used to calculate failure chance, remove this.
add skill_level_bonus to skill variable declaration in the beginning